JS/dom.js

```diff
@@ -0,0 +1,170 @@
+
+const header = document.querySelector('header .header-nav');
+let plusOneBtn = document.querySelector('.PlusOne');
+let minusOneBtn = document.querySelector('.MinusOne');
+let quantitySpan = document.getElementById('quantitySpan');
+let smallImages = document.querySelectorAll('.product-small-image');
+let mainImage = document.querySelector('.product-image');
+let mainDiv = document.querySelector('.product-popup');
+let xButton = document.querySelector('.x');
+let priceSpan = document.querySelector('.price');
+let quantityBase = 1;
+let descriptionSpan = document.querySelector('.description');
+let titleSpan = document.querySelector('.product-title');
+let countSpan = document.querySelector('.countSpan')
+const ArrayofCartObjects = JSON.parse(localStorage.getItem('ArrayofCartObjects') || '[]');
+
+// Window Events 
+window.onscroll = () => {
+    if (window.scrollY > 70) header.classList.add('active')
+    else header.classList.remove('active')
+}
+
+window.onload = () => {
+    if (window.scrollY > 70) header.classList.add('active')
+    else header.classList.remove('active')
+    RenderProducts();
+    let usrType = localStorage.getItem('usertype')
+    if (usrType == null) {
+        localStorage.setItem('usertype', 'client')
+        usrType = 'client'
+    }
+    // userTypeSwitcher.value = usrType
+    if (usrType == 'client') {
+        document.querySelectorAll(`.client`).forEach(e => e.style.display = 'inline-block')
+        document.querySelectorAll(`.seller`).forEach(e => e.style.display = 'none')
+    }
+    else if (usrType == 'seller') {
```

Comment:
It is better to define the two types of users you have in constants, for example: 
```js
const CLIENT = 'client'
const SELLER = 'seller'

if (usrType === CLIENT) {
```